app/javascript/utils/types.jsx

```diff
@@ -9,6 +9,32 @@ export type Scope = {
   botId: number
 };
 
+export type Translation = {
```

Comment:
I imagine this definitions would live in schema.json since they are part of the api that is exposed.
Let's keep track of that as technical debt and move forward.